website/docs/tutorials/07-testing/02-cheatcodes/assume.md

```diff
@@ -0,0 +1,25 @@
+# `assume`
+```python
+def assume(condition: bool) -> None:
+```
+`assume(condition)` works similarly to `assert condition` but instead of marking the example as failed it marks it as bad, thus preventing fuzz tests from using it as a falsifying example.
+
```

Comment:
- **it** marks **it**
- it looks similar to hypothesis docs however without context this description is hard to interpret
-  works similarly to `assert condition` — in Cairo asserts are used a bit differently

```suggestion
`assume(condition)` skips testing against given fuzzing arguments if the `condition` is `False`.
```